gui/src/components/mainInput/TipTapEditor.tsx

```diff
@@ -327,31 +327,44 @@ const TipTapEditor = memo(function TipTapEditor({
             props: {
               handleDOMEvents: {
                 paste(view, event) {
-                  console.log("Pasting image");
                   const items = event.clipboardData.items;
-                  for (const item of items) {
-                    const file = item.getAsFile();
-                    file &&
-                      modelSupportsImages(
+
+                  const hasImageItem = Array.from(items).some(
+                    item => item.type.startsWith('image/')
+                  );
+
+                  // Only log and process if we actually have an image
+                  if (hasImageItem) {
+                    console.log("Pasting image");
```

Comment:
console log!

Reply:
This was already in the code. I only moved it to inside this logic. @Fryingpannn do you want me to delete it?